src/gfn/states.py

```diff
@@ -737,9 +744,13 @@ def tensor(self) -> GeometricBatch:
         """
         if self.data.size == 0:
             dummy_graph = GeometricData(
-                x=torch.zeros(0, self.num_node_classes, device=self.device),
+                x=torch.zeros(
```

Comment:
@younik @hyeok9855 I wasn't entirely sure if `x` and `edge_attr` should *always* be either `float` or `int`...

Reply:
neither, I believe, so this change looks good

Reply:
ok actually, they're always a type of int now. we will need to add in the ability to make edge / node features continuous - issue filed here https://github.com/GFNOrg/torchgfn/issues/356